### PR TITLE
[merged] version: fix atomic version error

### PR DESCRIPTION
**Pull request by yuqi-zhang**
Add extra check to prevent errors for image IDs without repo name.
Before, when a user invokes "atomic version" on such an ID, it
returns "List index out of range" instead of label information
that exists.

Signed-off-by: Yu Qi Zhang <jerzhang@redhat.com>

### CODE REVIEW

File: Atomic/atomic.py [hunk 1]
```diff
@@ -362,7 +362,9 @@ def gen_cmd(self, cargs):
             return " ".join(args)
 
     def get_fq_name(self, image_info):
-        if len(image_info['RepoTags']) > 1:
+        if not image_info['RepoTags']:
+            return None
+        elif len(image_info['RepoTags']) > 1:
```
Comment (rhatdan) [inline Atomic/atomic.py]: I would prefer:
```
if image_info['RepoTags'] and len(image_info['RepoTags']) > 1:
```

Comment (yuqi-zhang) [inline Atomic/atomic.py]: If it doesn't exist it will default to:
`else:
    return image_info['RepoTags'][0]`
Which will also cause the issue.

Comment (rhatdan) [inline Atomic/atomic.py]: Ok no need of the else clauses.

```
        if not image_info['RepoTags']:
            return None

        if len(image_info['RepoTags']) > 1:
            if self.image in image_info['RepoTags']:
                return self.image

            raise ValueError("\n{} is tagged with multiple repositories. "
                             "Please use a repository name instead.\n".format(self.image))

        return image_info['RepoTags'][0]
```

Comment (rhatdan) [inline Atomic/atomic.py]: Not a big fan of elif clauses.

Comment (yuqi-zhang) [inline Atomic/atomic.py]: Sorry, updated.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**